contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -2679,6 +2679,8 @@ rewriteBatchLevelStatement(
 				auto commas = cctx->COMMA();
 				GetTokenFunc<TSqlParser::Function_optionContext*> getToken = [](TSqlParser::Function_optionContext* o) { return o->SCHEMABINDING(); };
 				bool all_removed = removeTokenFromOptionList(expr, options, commas, ctx, getToken);
+				getToken = [](TSqlParser::Function_optionContext* o) { return o->execute_as_clause()->CALLER(); };
```

Comment:
This condition will crash the server in few cases. 
First add a check to verify if `execute_as_clause()` exists, then `{ return o->execute_as_clause()->CALLER(); }` makes sense.

Reply:
Fixed in next revision by adding a pre-check for execute_as_clause().